src/lib.rs

```diff
@@ -70,6 +70,33 @@ pub struct Project {
     pub dynamic: Option<Vec<String>>,
 }
 
+impl Project {
+    /// Like `#[derive(Default)]`, but `name` is mandatory in PEP 621
+    pub fn default_with_name(name: String) -> Self {
```

Comment:
Wouldn't `new` be a more customary name for such a constructor method?

Reply:
makes more sense, yeah